.prettierrc.json

```diff
@@ -11,4 +10,4 @@
       }
     }
   ]
-}
+}
```

Comment:
I like newlines at end of files :)
(it's a unix convention, it makes text formatting easier for some tools)

Reply:
There's also a setting dedicated to this in vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Hmm, weird that I have to change settings for this.. We may not use the same formatter ? 🤔